js/src/features/project/components/Team/TeamChangeStatus.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Button } from "antd";
+
+import "./TeamChangeStatus.scss";
+
+type Props = {
+  onChange: () => void;
+  description: string;
+  btnText: string;
```

Comment:
That is not sense to make words shorter. Just `buttonText`.
You don't cut `description` to 'descr`, right?

Reply:
fix